.swiftlint.yml

```diff
@@ -6,6 +6,7 @@ disabled_rules:
   - opening_brace
   - valid_docs
   - type_name
+  - vertical_whitespace
```

Comment:
In my environment, `vertical_whitespace` rule generates warnings in Assembler.swift and Container.swift with Swiftlint v0.13.0. I thought it was ok to have 2 or more consecutive line breaks, so I excluded the rule rather than fixing the source code. I actually have no preference about this. If you like the rule, I'll remove excluding the rule and fix the source code.

Basically I agree that tests are also our codebase, but I excluded the tests from linting because we use Quick and our unit tests (or specs) have long `spec` functions. This is not my strong decision. It's ok to add a new issue to quit excluding the tests and fix some parts of .swiftlint.yml or test code.